src/server/api/controllers/delete-specific-batch.controller.js

```diff
@@ -0,0 +1,19 @@
+const knex = require('../../config/db');
+
+const deleteSpecificBatchById = async (batchId) => {
+  try {
+    await knex('batches_table')
```

Comment:
Is it a right name of the table? Should not be just 'batches'?

Reply:
yes it should be batches